RevenueCatUI/Helpers/PaywallData+Default.swift

```diff
@@ -18,7 +18,11 @@ extension PaywallData {
     public static let `default`: Self = .init(
         template: .multiPackageBold,
         config: .init(
-            packages: [.weekly, .monthly, .annual],
+            packages: [
+                Package.string(from: .weekly)!,
+                Package.string(from: .monthly)!,
+                Package.string(from: .annual)!
+            ],
```

Comment:
In a separate PR I'm going to change this to actually get the packages from the displayed Offering.